crm/optimize/outlook_extension.rst

```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and does not require any purchase to be installed or to create entries in
+your database.
+
+However, this extension provides you with *Lead Enrichment* in the **Company Insights** section.
+This service is part of a paid service named **Lead Generation**.
```

Comment:
Might be good to say...

This service is part of a paid service known as **Lead Generation**.

Reply:
To make suggestions in GitHub, you can take advantage of the **+-** button and hit Preview to see what it does.

Reply:
> Might be good to say...
> 
> This service is part of a paid service known as **Lead Generation**.

did the modification. And indeed, you can use the +- button
![image](https://user-images.githubusercontent.com/46810843/99953407-7e6c7400-2d81-11eb-9499-f5e2867ed0d9.png)